pymongo/compression_support.py

```diff
@@ -13,33 +13,16 @@
 # limitations under the License.
 from __future__ import annotations
 
+import importlib.util
 import warnings
 from typing import Any, Iterable, Optional, Union
 
-from pymongo._lazy_import import lazy_import
 from pymongo.hello import HelloCompat
-from pymongo.monitoring import _SENSITIVE_COMMANDS
+from pymongo.helpers import _SENSITIVE_COMMANDS
 
-try:
-    snappy = lazy_import("snappy")
-    _HAVE_SNAPPY = True
-except ImportError:
-    # python-snappy isn't available.
-    _HAVE_SNAPPY = False
-
-try:
-    zlib = lazy_import("zlib")
-
-    _HAVE_ZLIB = True
-except ImportError:
-    # Python built without zlib support.
-    _HAVE_ZLIB = False
-
-try:
-    zstandard = lazy_import("zstandard")
-    _HAVE_ZSTD = True
-except ImportError:
-    _HAVE_ZSTD = False
+_HAVE_SNAPPY = importlib.util.find_spec("snappy") is not None
+_HAVE_ZLIB = importlib.util.find_spec("zlib") is not None
+_HAVE_ZSTD = importlib.util.find_spec("zstandard") is not None
```

Comment:
Let's avoid using importlib. How about something like this instead?:
```python
def _have_snappy():
    try:
        import snappy
        return True
    except ImportError:
        return False
```

Reply:
Done